packages/node_modules/@webex/plugin-meetings/src/member/index.js

```diff
@@ -0,0 +1,174 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+import {
+  MEETINGS,
+  IN_LOBBY,
+  NOT_IN_MEETING,
+  IN_MEETING
+} from '../constants';
+
+import MemberUtil from './util.js';
+
+/**
+ * @class Member
+ */
+export default class Member {
+  namespace = MEETINGS;
+
+  /**
+   * @param {Object} participant - the locus participant
+   * @param {Object} options - optional constructor params
+   * @returns {Member}
+   * @memberof Member
+   * shallow copies locus participant object into a meeting member object
+   */
+  constructor(participant, options = {}) {
+    this.participant = null;
+    this.id = null;
+    this.name = null;
+    this.isAudioMuted = null;
+    this.isVideoMuted = null;
+    this.isSelf = null;
+    this.isHost = null;
+    this.isGuest = null;
+    this.isInLobby = null;
+    this.isInMeeting = null;
+    this.isNotInMeeting = null;
+    this.isNotAdmitted = null;
+    this.isContentSharing = null;
+    this.status = null;
+    this.isDevice = null;
+    this.isUser = null;
+    this.associatedUser = null;
+    this.isRecording = null;
+    this.processParticipant(participant);
+    this.processParticipantOptions(participant, options);
+  }
+
+  processParticipant(participant) {
+    this.participant = participant;
+    if (participant) {
+      this.processId(participant);
```

Comment:
I don't know if these need to be separate methods since each one does a single operation. Feels like you could do

```
this.id = MemberUtils.extractId(participant)
```

to slim down on the file and code hops.

Reply:
ah yeah i just wanted to keep it consistent for all the other process, got OCD, but I'll change, I could remove all of them though